lib/bls_keystore.go

```diff
@@ -0,0 +1,147 @@
+package lib
+
+import (
+	"github.com/deso-protocol/core/bls"
+	"github.com/deso-protocol/core/consensus"
+	"github.com/pkg/errors"
+)
+
+// BLSSigner is a wrapper for the bls.PrivateKey type, which abstracts away the private key
```

Comment:
this file seems like it belongs in the bls package

Reply:
So the idea with the BLSSigner is that it's a context-specific wrapper around the BLS utils. The BLSigner enumerates all the possible BLS signatures a DeSo node can make. Maybe we should consider renaming the struct/file to make it more protocol-specific. Like DeSoBLSSigner or something like that.

Reply:
+1 to @AeonSw4n's point. This struct is a protocol aware signer as opposed to a general purpose BLS utility. IMO, it's best placed in the lib package.

Reply:
okay cool!